src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/ErrorListInProcess.cs

```diff
@@ -122,7 +121,7 @@ private static string GetMessage(ITableEntryHandle item)
                 var unknown => unknown.ToString(),
             };
 
-            var message = $"({source}) {document}({line + 1}, {column + 1}): {severity} {errorCode}: {text}";
```

Comment:
Why was this changed?

Reply:
'source' is not a thing anymore.  we previously placed it in this info just so we could distinguish if we made the error item based on live items vs build items.